scripts/enable-resource-locking.sh

```diff
@@ -4,13 +4,16 @@ set -e
 SUBS=$1
 
 echo "retrieve all resource groups in a subscription"
-RESOURCE_GROUPS=$(az group list | jq '.[].name')
+RESOURCE_GROUPS=$(az group list --subscription DCD-CFTAPPS-SBOX | jq '.[].name')
 
 for rg in ${RESOURCE_GROUPS[@]}; 
 do
   RG_NAME=$(sed -e 's/^"//' -e 's/"$//' <<<"$rg")
+  RSRC_LIST=$(az resource list -g $RG_NAME --subscription DCD-CFTAPPS-SBOX | jq '.[].type' | sed -e 's/^"//' -e 's/"$//' | grep -ciE 'Storage|KeyVault|Sql')
```

Comment:
can't you use a query rather than jq + sed + grep?

Reply:
added a JSONPATH for retrieving the resource group list enforcing locks.